docs/include/list-table-parts.rst

```diff
@@ -4,8 +4,6 @@
    usage: pgcopydb list table-parts  --source ... 
    
      --source                    Postgres URI to the source database
-     --force                     Force fetching catalogs again
-     --schema-name               Name of the schema where to find the table
-     --table-name                Name of the target table
```

Comment:
I think we still want to keep these options, even if we now have support for running the command without using the options.